src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -553,6 +553,9 @@ message BidExt {
         // DSP provided impression tracking pixels. If the request indicates a
         // secure environment via the imp.secure flag, all URLs must be secure.
         repeated string imptrackers = 14;
+
+        // Temporary field of pmptier to allow for transition period
+        optional int32 pmptier = 15;
```

Comment:
the comment doesn't seem accurate anymore. Would be good to add the same detail as the `deprecated_pmptier` comment for this field.